test_runner/fixtures/utils.py

```diff
@@ -37,6 +37,23 @@
 
 
 Fn = TypeVar("Fn", bound=Callable[..., Any])
+COMPONENT_BINARIES = {
+    "storage_controller": ("storage_controller"),
+    "storage_broker": ("storage_broker"),
+    "compute": ("compute_ctl"),
+    "safekeeper": ("safekeeper"),
+    "pageserver": ("pageserver", "pagectl"),
+}
+# Disable auto-formatting for better readability
+# fmt: off
```

Comment:
You need to reenabled it back after the block

Reply:
done